crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -9,6 +9,67 @@ Changelog - nrf_cc3xx_platform
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_platform - 0.9.16
+***************************
+
+New version of the library with the following features:
+
+* Added code-size optimized API for SHA-256 calculation in RAM and flash.
+* Fixed a bug where platform mutexes may end up unallocated if a context holding it is not zeroized.
+
+Library built against Mbed TLS version 3.1.0.
+
+Added
+=====
+
+Added a new build of nrf_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and nRF5340.
+
+.. note::
+
+   The *short-wchar* libraries are compiled with a wchar_t size of 16 bits.
+
+* nrf_cc312_platform, nRF5340 variants
+
+  * ``cortex-m33/hard-float/libnrf_cc312_platform_0.9.16.a``
+  * ``cortex-m33/soft-float/libnrf_cc312_platform_0.9.16.a``
+
+  * No interrupts
+
+    * ``cortex-m33/soft-float/no-interrupts/libnrf_cc312_platform_0.9.16.a``
+    * ``cortex-m33/hard-float/no-interrupts/libnrf_cc312_platform_0.9.16.a``
+
+  * short-wchar
+
+    * ``cortex-m33/hard-float/short-wchar/libnrf_cc312_platform_0.9.16.a``
+    * ``cortex-m33/soft-float/short-wchar/libnrf_cc312_platform_0.9.16.a``
+
+  * short-wchar, no interrupts
+
+    * ``cortex-m33/hard-float/short-wchar/no-interrupts/libnrf_cc312_platform_0.9.16.a``
+    * ``cortex-m33/soft-float/short-wchar/no-interrupts/libnrf_cc312_platform_0.9.16.a``
```

Comment:
Let's be consistent with the other library and use **

Reply:
Will do this